streaming_main.py

```diff
@@ -523,6 +533,24 @@ def main() -> None:
         help="ignore tests starting with test/built-ins/RegExp/ since they can take up to 80% of total runtime",
     )
 
+    parser.add_argument(
+        "--lists",
+        default=-1,
```

Comment:
These options can be useful to test if we are optimally running the tests.
Since this might be different for other computers with different core counts etc.